models/ac_generator.cpp

```diff
@@ -41,6 +43,12 @@
 
 namespace nest
 {
+void
+register_ac_generator()
+{
+  kernel().model_manager.register_node_model< ac_generator >( "ac_generator" );
+}
```

Comment:
I think this would be a good time to create a wrapper for `register_node_model()` in `nest.h`/nest_impl.h` as we already have it for `register_connection_model()`. I guess that we could then also replace most of the includes above by just `nest_impl.h`

Reply:
Indeed.